packages/grpc-js-core/src/channel.ts

```diff
@@ -224,7 +225,14 @@ export class Http2Channel extends EventEmitter implements Channel {
     Promise.all([finalMetadata, this.connect()])
       .then(([metadataValue]) => {
         let headers = metadataValue.toHttp2Headers();
-        headers[HTTP2_HEADER_AUTHORITY] = this.authority.hostname;
+        let host: string;
+        // TODO(murgatroid99): Add more centralized handling of channel options
+        if (this.options['grpc.default_authority']) {
+          host = this.options['grpc.default_authority'] as string;
```

Comment:
Can you add this as an entry in the type def? https://github.com/grpc/grpc-node/pull/209/files#diff-6270a066e64c77ba494d58b714011deeL40